spec/models/participant_declaration_spec.rb

```diff
@@ -436,4 +436,25 @@
       end
     end
   end
+
+  describe "#voidable?" do
+    [
+      { state: "submitted", voidable: true },
+      { state: "eligible", voidable: true },
+      { state: "payable", voidable: true },
+      { state: "paid", voidable: false },
+      { state: "voided", voidable: false },
+      { state: "ineligible", voidable: true },
+      { state: "awaiting_clawback", voidable: false },
+      { state: "clawed_back", voidable: false },
+    ].each do |hash|
+      context "when declaration is #{hash[:state]}" do
```

Comment:
not a big fan of dynamically generated examples, becomes a bit tricky if you need to debug one in the middle. 
Would you be able to example those? Thanks.

Reply:
if you need to debug a single failing test you can change/filter the array to the exact one test you want to run. alternatively you can simply create a new test with the exact setup you want to test.